kubernetes/nginx/README.md

```diff
@@ -96,6 +107,13 @@ Secrets.
 
 # Create Nginx Docker Image
 
+We currently deploy a custom Nginx docker image that includes the
+OpenWhisk CLI and other downloadable artifacts. Once there are proper
+releases of these artifacts, we can switch to using a standard Nginx
+image and redirect to the official release archives for the artifacts
+we are currently storing in the custom docker image.  See the GitHub
+[issue](https://github.com/openwhisk/openwhisk/issues/2152).
```

Comment:
I think all the bundled artifacts are now externally released.

Reply:
Edit: nope, the CLI is still copied in instead of linking to the release.

@csantanapr @houshengbo is this planned for future?